src/components/cms/SkillPage/SkillListing.js

```diff
@@ -359,7 +359,7 @@ class SkillListing extends Component {
                       key={index}
                       data={data}
                       history={history}
-                      margin={'1.5% 2.85% 1.5% 0'}
+                      margin={'12px 10px 12px 0'}
```

Comment:
Why `pixles` insted of `percentage` ?

Reply:
Percentage is a relative unit. So, the margin will change according to the screen of size.On a really small screen like a phone that might be appropriate. But if you're viewing that same page on a bigger size screen, that margin is going to be huge if we use percentage. That's why I've used pixels to maintain the same gap between them.